### ISSUE TITLE
Duplicate id on Salary field

### ISSUE THREAD

**Issue by GregoryForrest** [OZI-SP-Dev/rpa-tracker#11]
The salary range inputs have the same Id.
The Err ids should also be updated to identify one over the other
An aria attribute should be added to identify the min and max salary inputs so a screen reader can identify what value would be for which field
